README.md

```diff
@@ -290,12 +290,12 @@ These configuration elements are enough for most connection scenarios. Now we wi
 
 ### 1.4.2. Configuring Hazelcast Python Client
 
-Hazelcast Python client can be configured programmatically.
+The Hazelcast Python client can be configured programmatically.
 
 This section describes some network configuration settings to cover common use cases in connecting the client to a cluster. See the [Configuration Overview section](#3-configuration-overview)
 and the following sections for information about detailed network configurations and/or additional features of Hazelcast Python client configuration.
 
-An easy way to configure your Hazelcast Python client is to create a `ClientConfig` object and set the appropriate options. Then you can
+An easy way to configure your Hazelcast Python client is to create a `ClientConfig` object and set the appropriate config options. Then you can
```

Comment:
An easy way to configure your Hazelcast Python client is to create a `ClientConfig` object and set the appropriate configuration options. Then you can
supply this object to your client at the startup. This is the programmatic configuration approach.